media-video/mjpg-streamer/files/mjpg-streamer.initd

```diff
@@ -34,8 +35,8 @@ start() {
 		--user "${MJPG_STREAMER_USER}" \
 		--group "${MJPG_STREAMER_GROUP}" -w 100 -b -m \
 		--pidfile "${MJPG_STREAMER_PIDFILE}" \
-		-- -i "/usr/lib/input_${INPUT_PLUGIN}.so ${INPUT_PLUGIN_OPTS}" \
-		-o "/usr/lib/output_${OUTPUT_PLUGIN}.so ${OUTPUT_PLUGIN_OPTS}"
+		-- -i "/usr/lib64/input_${INPUT_PLUGIN}.so ${INPUT_PLUGIN_OPTS}" \
+		-o "/usr/lib64/output_${OUTPUT_PLUGIN}.so ${OUTPUT_PLUGIN_OPTS}"
 	eend $? "Check syslog to see why startup failed."
```

Comment:
$(get_libdir) to get either lib or lib64 or what else depending on profile/arch

Reply:
done